gapis/api/gles/compat.go

```diff
@@ -562,6 +562,13 @@ func compat(ctx context.Context, device *device.Instance) (transform.Transformer
 				log.E(ctx, "Error decompressing texture: %v", err)
 			}
 
+		case *GlTexBufferEXT:
+			if !version.IsES { // Strip suffix on desktop.
```

Comment:
I have added it, but I worried about this exponentiation grow of different code paths we take